packages/block-editor/src/components/link-control/README.md

```diff
@@ -70,37 +152,50 @@ Whether to present initial suggestions immediately.
 - Type: `boolean`
 - Required: No
 
-If passed as either `true` or `false`, controls the internal editing state of the component to respective show or not show the URL input field.
+If passed as either `true` or `false`, controls the internal editing state of
+the component to respectively show or not to show the URL input field.
 
 
 ### createSuggestion
 
 - Type: `function`
 - Required: No
+- Returns: when called may return either a new `suggestion` directly or a `Promise` which resolves to a
+new `suggestion`.
+
+Used to handle the dynamic creation of a new `suggestion` (and ultimately new link
+`value`) within the Link UI.
 
-Used to handle the dynamic creation of new suggestions within the Link UI. When
-the prop is provided, an option is added to the end of all search
-results requests which when clicked will call `createSuggestion` callback
-(passing the current value of the search `<input>`) in
-order to afford the parent component the opportunity to dynamically create a new
-link `value` (see above).
+When provided, an option is appended to all search
+results requests which when clicked will call the `createSuggestion` callback
+(passing the current value of the search `<input>`). This affords the parent component the opportunity to dynamically create a new
+link `suggestion` (see above).
 
-This is often used to allow on-the-fly creation of new entities (eg: `Posts`,
-`Pages`) based on the text the user has entered into the link search UI. For
-example, the Navigation Block uses this to create Pages on demand.
+This `suggestion` will then be _automatically_ passed to the
+`onChange` handler to create **the next link value**.
 
-When called, `createSuggestion` may return either a new suggestion directly or a `Promise` which resolves to a
-new suggestion. Suggestions have the following shape:
+As a result of the above, this prop is often used to allow on-the-fly creation of new entities (eg: `Posts`,
+`Pages`) based on the text the user has entered into the link search UI. As an
+example, the Navigation Block uses `createSuggestion` to create Pages on fly
```

Comment:
"to create Pages on fly" -> "to create Pages on-the-fly"

Reply:
Nice spot. Fixed.